### ISSUE TITLE
FALSE POSITIVE -  databricks.com  (TRACKING LIST)

### ISSUE THREAD

**Issue by AyanokojiK1yotaka** [ShadowWhisperer/BlockLists#153]
databricks.com

**Comment by ShadowWhisperer** [comment]
Databricks, Inc. is a global data, **analytics** and artificial intelligence company founded by the original creators of Apache Spark. The company provides a cloud-based platform to help enterprises build, scale, and govern data and AI, including generative AI and other machine learning models. - _Wikipedia_

Which site does this break ?

**Comment by AyanokojiK1yotaka** [comment]
i cant access the original site

**Comment by AyanokojiK1yotaka** [comment]
Databricks is a unified, open analytics platform that helps organizations build, deploy, share, and maintain enterprise-grade data, analytics, and AI solutions at scale. Key points about Databricks:
It integrates with cloud storage and security in the customer's cloud account (Azure, AWS, GCP) and manages the cloud infrastructure on their behalf.
The platform uses generative AI to understand the semantics of the customer's data, automatically optimize performance, and manage infrastructure.
It provides a workspace with a unified interface for data processing, ETL, generating dashboards, managing security and governance, data exploration, machine learning modeling, and generative AI solutions.
Databricks has a strong commitment to open source and manages updates of open source integrations in its Databricks Runtime releases.
It allows customers to access data in their own data stores using their own credentials, without forcing data migration into proprietary storage.
Common use cases include building a data lakehouse, data warehousing and BI, data governance and secure data sharing, DevOps and CI/CD, ETL and data engineering, and machine learning/AI.
In summary, Databricks offers a comprehensive, cloud-based platform to help enterprises derive value from their big data through analytics, machine learning and AI, while managing the underlying infrastructure and security.

**Comment by ShadowWhisperer** [comment]
Whitelisted

**Comment by ShadowWhisperer** [comment]
After doing some more research into this company, I see no reason for it to be allowed; unless you work for them. I suggest you whitelist it on your end. Will be stay blocked, here.
